LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
This is what `reuse download MIT` placed there automatically. I think it makes sense that it's blank because many files in the project can refer to one license but with different years and copyright holders.